src/DbSelect.php

```diff
@@ -161,10 +163,11 @@ public function getArrayCopy()
     }
 
     /**
+     * @param  array $row
      * @return int
```

Comment:
These two lines can be removed because the method signature already gives the same information.